R/percent.R

```diff
@@ -0,0 +1,210 @@
+#' Percentages
+#'
+#' @description
+#'
+#' `percent` is a lightweight S3 class allowing for pretty
+#' printing of proportions as percentages. \cr
```

Comment:
```suggestion
#' printing of proportions as percentages.
```

Reply:
Is it the entire description you want as just that 1 line? Not sure it fully reflects why users might want to use percents.

Reply:
Sorry, my 'suggestion' was just to remove the `\cr` which I assume is a typo?

Reply:
Ah, no I typically use it to add a new line, but maybe there's better syntax for that.

Reply:
The `\cr` appears verbatim in the .Rd file (https://github.com/Public-Health-Scotland/phsmethods/blob/b9a9151ab0ab2a89e8f3527b929c2ab4e93f60f9/man/percent.Rd#L17). I would just put a new line in, no special characters needed.

Reply:
Maybe an obvious one but how do I get the documentation to display that line on a new line right below the previous text? If I insert a new line using Enter on my keyboard it places it 2 lines below.